Cargo.toml

```diff
@@ -18,8 +18,19 @@ readme = "README.md"
 categories = ["algorithms", "science"]
 keywords = ["beta", "error", "gamma"]
 
+[features]
+default = ["system_math"]
+
+# Activating feature "system_math" (which is the default) instructs `special` to
+# use the system math C library for mathematical operations. Deactivate this
+# default feature if no system math C library is available on your compilation
+# target (e.g., if you compile to WebAssembly, or if you want test for undefined
+# behaviour using `miri`).
+system_math = ["libc"]
+
 [dependencies]
-libc = "0.2"
+libc = {version = "0.2", optional = true} 
+libm = "0.2.2" # Only used if default feature "libc" is *not* activated.
```

Comment:
Should actually read: `# Only used if default feature "system_math" is *not* activated.`